com.leohanney.Volaris.yml

```diff
@@ -0,0 +1,38 @@
+id: com.leohanney.Volaris
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+          
+command: volaris-gui
+rename-icon: com.leohanney.Volaris
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --share=ipc
+
+          
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/volar-is/volaris-gui/releases/download/app-v0.1.0/Volaris_0.1.0_amd64.deb
+        sha256: c55febeee6cbde904b52119e86606dd08eb474aa8fc53be16843841334cdcc3c
+        only-arches: [x86_64]
+      - type: file
+        url: https://raw.githubusercontent.com/volar-is/volaris-gui/main/flatpak.xml
+        sha256: 61ce16a64ab7afb0221f0d943bd381817a6b5617e993106003ca899fa9ede748
+                  
+    build-commands:
+       - ar -x *.deb
+       - tar -xf data.tar.gz
+       - mkdir -p /app/bin
+       - cp -r usr/bin/* /app/bin
+       - sed -i 's/Categories=/Categories=Utility/' usr/share/applications/volaris-gui.desktop
+       - sed -i 's#Comment=A Tauri App#Comment=Volaris GUI is a sleek Svelte/Tauri-based app for easy file encryption and decryption using key files.#' usr/share/applications/volaris-gui.desktop
```

Comment:
use patches or desktop-file-edit instead of sed

Reply:
noted, will do that

Reply:
note that "Svelte/Tauri" has nothing to do in that field. This is technobabble no user care about.

Reply:
removed that